capstone/src/main/java/com/google/univiz/CollegeDataApi.java

```diff
@@ -0,0 +1,9 @@
+package com.google.univiz;
+
+import java.util.List;
+
+/** Provides access to college related data. */
+public interface CollegeDataApi {
```

Comment:
It looks like other api interfaces are using the `{FeatureName}Resource` naming convention. Do we want to use the same convention for this interface too: `CollegeDataResource` or this is different because it's not tied to a particular feature?

Reply:
not sure. it isn't tied to a particular feature, i think its just responsible for REST requests.

Reply:
IIUC the `**Api` classes are to be implemented inside the scorecard package. The `**Resource` classes will inject the `**Api` classes, and the servlet will inject the ``**Resource` classes. So I think this is ok to stay as `CollegeDataApi`. 

Does that match what everyone else is expecting?

Reply:
yes i think so.